contracts/modules/capital/Pool.sol

```diff
@@ -230,15 +230,23 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
     bool ok;
     address asset = assets[assetId];
 
+    // [todo] consider adding a minimum sum for triggering an MCR update to save gas
+    uint amountInETH;
     if (asset == ETH) {
       // solhint-disable-next-line avoid-low-level-calls
       (ok, /* data */) = payoutAddress.call.value(amount)("");
+      amountInETH = amount;
     } else {
-      ok =  _safeTokenTransfer(assets[assetId], payoutAddress, amount);
+      ok =  _safeTokenTransfer(asset, payoutAddress, amount);
+      uint rate = priceFeedOracle.getAssetToEthRate(asset);
+      require(rate > 0, "Pool: Zero rate");
+      amountInETH = amount.mul(rate).div(1e18);
     }
 
     if (ok) {
       emit Payout(payoutAddress, asset, amount);
+      uint totalAssetValue = getPoolValueInEth();
+      mcr.updateMCRInternal(totalAssetValue.sub(amountInETH), true);
```

Comment:
you don't need to substract `amountInETH` here, `getPoolValueInEth()` should include it